Create Meet the Hoodies: Gregor blog post!

jennwrites opened the pull request:
fixes hoodiehq/editorial#74

I stubbed out this blog post but there's still things to be done here for @kmcrayton7 :relaxed:
- [x] Add an intro about interviewing Gregor to the top of the blog post
- [x] Find the link to the sloth image so we can include that
- [x] Address any feedback in the review/PR process 

Please ping me if you need any assistance!

jennwrites committed:
Create 2016-04-14-meet-the-hoodies-gregor-martynus.md

gr2m committed:
minutes.io with lowercase m

gr2m commented on _posts/2016-04-14-meet-the-hoodies-gregor-martynus.md:
```diff
@@ -0,0 +1,42 @@
+
+## 3. What’s your favorite use of Hoodie in the wild?
+
+[minutes.io](https://minutes.io/welcome). It has 20,000 active users helping to make sure that Hoodie is based on ‘real’ user experiences. minutes.io is to Hoodie as Basecamp is to Rails.
```
better say thousands of active users

kmcrayton7 committed:
Update 2016-04-14-meet-the-hoodies-gregor-martynus.md

Requested changes to "Create Meet the Hoodies: Gregor blog post! #274" have been made

gr2m committed:
Meet the Hoodies: Gregor - remove sloth image

license is unclear, and youtube video is good enough

kmcrayton7 committed:
Added files via upload

Image of sloth

kmcrayton7 committed:
Update 2016-04-14-meet-the-hoodies-gregor-martynus.md

Added a new image of a sloth

gr2m committed:
fixed image path

gr2m committed:
fixed image alt text

gr2m committed:
remove unneeded `

gr2m committed:
minor fixes

gr2m committed:
Sloths have super powers!

gr2m commented:
LGTM now :)

varjmes commented on _posts/2016-04-14-meet-the-hoodies-gregor-martynus.md:
```diff
@@ -0,0 +1,42 @@
+
+minutes.io is an application I built which allows for meeting minutes to be captured in an easy and effective way. While attending the 2014 CouchConf in Berlin (because minutes.io uses [CouchDB](https://cwiki.apache.org/confluence/display/COUCHDB/Introduction), a NoSQL database) I met [Jan Lehnardt](https://twitter.com/janl). He was interested in further building out the application, so he became ...
+
+In an effort not to become attached to a name, the application was initially named CANG (Code Application Next Generation). Which was the worse name we could think of at the time. But once we realized that we had a viable product the name search began. After some time, we were sitting around and Jan pulled the hood of his sweatshirt onto his head and said, “Why not call it hoodie”? Our mission ha ...
```
"was the worse" should be "was the worst"

varjmes commented:
Just one little comment I noticed.

Also, awesome to finally hear how the name 'Hoodie' came to be!

schisepo commented:
LGTM - Like Charlotte, I love how the name Hoodie came about. Pretty Cool!

jennwrites commented:
LGTM, great work Kim and Gregor!

jennwrites commented:
Let's plan on publishing this Thursday, April 28!

schisepo commented:
Looks good to me ☺ here's a link to a card with the blog post announcement for Twitter: https://trello.com/c/4RloA9TQ/7-meet-the-hoodies
